test/integration/targets/ios_acl_interfaces/tests/cli/merged.yaml

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
Please add assertions for before/after dicts in all the test.

Reply:
IMO it's not required to specifically check for before and after dicts generated if facts is working as expected and commands fired are correct.